### PR TITLE
Add interface declaration for new pixel event vtex:newsletterSubscription

### PR DESCRIPTION
**Pull request by luizgamabh**
**What problem is this solving?**

Add interface declaration for new pixel event vtex:newsletterSubscription

**How should this be manually tested?**

Not applicable, but all fields are compatible with event data

**Screenshots or example usage:**

Not applicable

### CODE REVIEW

File: react/typings/events.d.ts [hunk 1]
```diff
@@ -161,6 +162,14 @@ export interface PromotionClickData extends EventData {
   promotions: Promotion[]
 }
 
+export interface NewsletterSubscriptionData extends Exclude<EventData, 'currency'> {
```
Comment (kaisermann) [inline react/typings/events.d.ts]: Why are you excluding the currency prop?

**Review by kaisermann — CHANGES_REQUESTED** [review]

File: react/typings/events.d.ts (hunk 1 above)
Comment (gutogama) [reply react/typings/events.d.ts]: I figured it wouldn't make sense in the context of the newsletter event, but I can keep the property.

**Comment by Mogueno** [comment]
@kaisermann Hello!
Can you please review this PR? 
 I belive that the fix on the interface has been done.

**Review by kaisermann — APPROVED** [review]

**Comment by Mogueno** [comment]
@kaisermann Fixed the lint problem, I believe it's ready to merge!

**Comment by Mogueno** [comment]
@victorhmp Fixed the lint problem, I believe it's ready to merge!

**Review by victorhmp — APPROVED** [review]

**Pull request merged**